app/src/main/java/de/danoeh/antennapod/adapter/PlaybackStatisticsListAdapter.java

```diff
@@ -0,0 +1,96 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.recyclerview.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+import de.danoeh.antennapod.core.util.Converter;
+
+/**
+ * Adapter for the playback statistics list
+ */
+public class PlaybackStatisticsListAdapter extends StatisticsListAdapter {
+
+    public PlaybackStatisticsListAdapter(Context context) {
+        super(context);
+    }
+
+    @Override
+    public int getItemCount() {
+        return statisticsData.feedTime.size() + 1;
+    }
+
+    public DBReader.StatisticsItem getItem(int position) {
+        if (position == 0) {
+            return null;
+        }
+        return statisticsData.feedTime.get(position - 1);
+    }
+
+    @NonNull
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(context);
+        if (viewType == TYPE_HEADER) {
+            View view = inflater.inflate(R.layout.statistics_listitem_total, parent, false);
+            TextView totalText = view.findViewById(R.id.total_description);
+            totalText.setText(R.string.total_time_listened_to_podcasts);
```

Comment:
If you extract a method `getHeaderCaption()`, this method can be moved to the parent class.